TheMET/TheMET/ArtView/ArtView.swift

```diff
@@ -76,6 +76,7 @@ class ArtView: UIView {
         self.loadingImageView.translatesAutoresizingMaskIntoConstraints = false
         self.likeButton.translatesAutoresizingMaskIntoConstraints = false
         self.textLabel.translatesAutoresizingMaskIntoConstraints = false
+//        tagView placement problem is going to be solved in  one the next tasks.
```

Comment:
Please provide the task-number.
Also you cause a `TODO:` marker to make it visible in code-hierarchy (just like `MARK:`)

Reply:
+